extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/GrpcServerRecorder.java

```diff
@@ -198,12 +222,13 @@ private void buildGrpcServer(Vertx vertx, GrpcServerConfiguration configuration,
         if (securityHandlers != null) {
             for (Map.Entry<Integer, Handler<RoutingContext>> e : securityHandlers.entrySet()) {
                 Handler<RoutingContext> handler = e.getValue();
+                boolean isAuthenticationHandler = e.getKey() == -200;
```

Comment:
What's this "-200" ?

Reply:
It is 200 * -1, see:

- https://github.com/quarkusio/quarkus/blob/a6413e331c3b072c9e624d93189772341d2547ad/extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/FilterBuildItem.java#L16
- https://github.com/quarkusio/quarkus/blob/a6413e331c3b072c9e624d93189772341d2547ad/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java#L430

I can imagine it feels wrong, but if I introduce runtime constant, this place will be only consumer anyway, so it won't give anyone hint that they need to update it in this class. FWIW changing priority would be super breaking, but I can make it constant if you like.

Reply:
Or give it order and know that the first handler is the authenticator. I think -200 is acceptable, but if you will different, just let me know and I'll rewrite it to avoid it entirely. np.